### ISSUE TITLE
Multiple containers, different versions, only one entry in Images tab

### ISSUE THREAD

**Issue by rasmuslp** [fmartinou/whats-up-docker#17]
Hi,

When I run two containers of the same image -- but different version ranges and include regexes -- only one entry shows in the Images tab. I would expect to see different updates available for the two containers. 
NB: This _might_ be the effect of #16 ?

However, what I would really like is a container based view. Which I believe makes sense, now that the include tags (and others) are specified per container.

**Comment by fmartinou** [comment]
Hi,

Definitely a bug :(

The expected behaviour is to get 1 entry for each unique couple `image+tag`.

Let me explain with this simple example:

```yml
version: '3'

services:
    nginx_1:
        image: nginx:1.9-alpine
        labels:
            - 'wud.tag.include=^[0-9]\d*\.[0-9]\d*-alpine$$'

    nginx_2:
        image: nginx:1.10-alpine
        labels:
            - 'wud.tag.include=^[0-9]\d*\.[0-9]\d*-alpine$$'

    nginx_3:
        image: nginx:1.10-alpine
        labels:
            - 'wud.tag.include=^[0-9]\d*\.[0-9]\d*-alpine$$'
```

So 3 nginx instances:
- One 1.9 instance
- Two 1.10 instances

I expect to see 2 entries in the images tab and to get 2 notifications:
- 1.9 update
- 1.10 update

Would it suit you?

**Comment by rasmuslp** [comment]
Thank you for the elaborate example. I believe that would work for my use case. Even though I use different `wud.tag.include` labels, I do so on containers running different `tag`s.

However, given how WUD allows configuration, then why not also include the `include`/`exclude` labels as part of that uniqueness, so that it would be `image+tag[+include/exclude]`?

**Comment by fmartinou** [comment]
Yes, it should be possible to include the `wud labels` to compute the uniqueness.
I just don't have in mind any valid use case where I'd want to track identical (image+tag) images with different `include/exclude` labels. 🤔 

Do you have an example to let me understand?

**Comment by rasmuslp** [comment]
No. I tried, but I could only come up with contrived examples 🙃 I thought it would make sense, but atm. I can't really see a real world scenario, where where it would be relevant.

**Comment by fmartinou** [comment]
Anyway, I must fix the issue regarding uniqueness so... if it' not too complicated, I can include the labels as well.

**Comment by rasmuslp** [comment]
Much appreciated :)

**Comment by fmartinou** [comment]
Hi,

It's normally fixed in version `3.1.1`.
